providers/keycloak/src/airflow/providers/keycloak/auth_manager/routes/login.py

```diff
@@ -70,3 +72,33 @@ def login_callback(request: Request):
     secure = bool(conf.get("api", "ssl_cert", fallback=""))
     response.set_cookie(COOKIE_NAME_JWT_TOKEN, token, secure=secure)
     return response
+
+
+@login_router.get("/refresh")
+def refresh(
+    request: Request, user: Annotated[KeycloakAuthManagerUser, Depends(get_user)]
+) -> RedirectResponse:
+    """Refresh the token."""
+    client = KeycloakAuthManager.get_keycloak_client()
+
+    if not user or not user.refresh_token:
+        raise HTTPException(
+            status_code=400, detail="User is not a valid Keycloak user or has no refresh token"
+        )
+
+    tokens = client.refresh_token(user.refresh_token)
+    userinfo = client.userinfo(tokens["access_token"])
```

Comment:
Do we need it? Does it change between tokens? We already have his information in `user` right?

Reply:
Normally, it doesn't. We can reduce one call by removing it. Great catch! :)